src/type/s2k.js

```diff
@@ -57,6 +57,23 @@ S2K.prototype.get_count = function () {
   return (16 + (this.c & 15)) << ((this.c >> 4) + expbias);
 };
 
+S2K.prototype.set_count = function (count) {
+  var expbias = 6;
+  
+  var upperBits = 0;
+  while (upperBits < 15 && count > (31 << (upperBits + expbias))) {
+    upperBits++;
+  }
+  
+  var lowerBits = 0;
+  while (lowerBits < 15 && count > (16 + lowerBits) << (upperBits + expbias)) {
+    lowerBits++;
+  }
+
+  this.c = lowerBits + (upperBits << 4);
```

Comment:
I honestly cannot judge with the naked eye if this is correct or not. Perhaps add some unit test with several test vectors and please document your reasoning behind the code.